requirements.txt

```diff
@@ -1,2 +1,5 @@
 six
+ansible
 ansible-runner
+paramiko
+ncclient
```

Comment:
I'm not sure we require ncclient?
I think we default to ssh for all connections.

Reply:
ugh, your right.  will update the PR